### PR TITLE
gitextensions: Update to version 4.0.0

### PR DESCRIPTION
**Pull request by raycheung**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by raycheung** [comment]
The checkver url is related to https://github.com/ScoopInstaller/Scoop/issues/5190 .

File: bucket/gitextensions.json [hunk 1]
```diff
@@ -22,10 +21,10 @@
     ],
     "persist": "GitExtensions.settings",
     "checkver": {
-        "github": "https://github.com/gitextensions/gitextensions",
+        "url": "https://github.com/gitextensions/gitextensions/releases/expanded_assets/v4",
```
Comment (rasa) [inline bucket/gitextensions.json:24]: I guess we have to hardcode the version here, right?

**Review by rasa — APPROVED** [review]

**Pull request merged**